addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -452,6 +452,13 @@ function makeActionManager(env) {
         if (view.type === "form") {
             if (action.target === "new") {
                 viewProps.mode = "edit";
+                if (!viewProps.onSave) {
+                    viewProps.onSave = (record, params) => {
+                        if (params && params.closable) {
```

Comment:
We had issues with the onSave being called by the viewButtonHook, closing the dialog when it shouldn't. So I added this parameter to only close the dialog from the action service when given